<launch>
<!--
	<node pkg="guidance" type="guidanceNode" name="guidanceNode" output="screen" />

	<include file="$(find dji_sdk)/launch/sdk_manifold.launch"/>
-->
<!--
	<node pkg="jet" type="sdk_odom_to_guidance_odom" name="sdk_odom_to_guidance_odom" output="screen" />
-->
	<include file="$(find jet)/launch/cfg.launch"/>
	<include file="$(find vision)/launch/vision.launch"/>

	<node pkg="jet" type="jet" name="jet" output="screen">
	<param name="spin_rate"           type="int"    value="50"/>
	<param name="serial_port"         type="string" value="/dev/ttyUSB1"/>  
  	<param name="serial_baudrate"     type="int"    value="115200"/>

	<param name="vision_target_pos_filter_window_size"   type="int"    value="5"/>
	<param name="vision_target_pos_filter_variance_limit"   type="double"    value="1e-5"/>

	<param name="use_guidance"        type="bool"   value="false" />
	
	<remap from="/odom_raw" to="/dji_sdk/odometry" />
	
	</node>

</launch>

